bin/build-languages.mjs

```diff
@@ -0,0 +1,246 @@
+import crypto from 'crypto';
```

Comment:
Note: git didn't detect that the file got renamed, so we need to make sure the old file is deleted.